### ISSUE TITLE
In Definitions "formatted as" cannot be set to string

### ISSUE THREAD

**Issue by 0815fox** [Apicurio/apicurio-studio#413]
When I go into definitions, create a property and set type to string and formatted as to string as well, after reloading, formatted as is empty ("Choose format") again.

The cause probably is, that the value string is optional as it is the default and thus the tool does not store it. However, from a usability point of view it is not intuitive. Maybe the ui should show the default value even in case it is not present?

**Comment by EricWittmann** [comment]
Yeah that's weird - and your suggestion is a good one.  Thanks for the report and the suggested fix!  Should be an easy change to make...
